rmw_opensplice_cpp/src/rmw_node_names.cpp

```diff
@@ -31,5 +31,4 @@ rmw_get_node_names(
   RMW_SET_ERROR_MSG("get_node_names is not supported for Opensplice");
   return RMW_RET_ERROR;
 }
-
```

Comment:
because cpplint raises "/home/mikael/work/ros2/ros2_ws/src/ros2/rmw_opensplice/rmw_opensplice_cpp/src/rmw_node_names.cpp:34:  Redundant blank line at the end of a code block should be deleted.  [whitespace/blank_line] [3]" otherwise